app/connectors/EmailClient.scala

```diff
@@ -126,4 +147,9 @@ object EmailDateFormatter {
     dateTime.toString("d MMMM yyyy, h:mma")
       .replace("AM", "am").replace("PM", "pm") // Joda time has no easy way to change the case of AM/PM
   }
+
+  def convertToHoursOrDays(timeUnit: TimeUnit, timeLeft: Int): String = {
```

Comment:
So timeLeft is always in hours? And this method converts? It was tough to understand, could you perhaps change the name of the 'timeLeft' parameter to 'timeLeftInHours'?